### ISSUE TITLE
Node.js support

### ISSUE THREAD

**Issue by matthew-a-thomas** [matthew-a-thomas/js-ldpc#5]
Turn this into something that will work on Node.js

**Comment by JakeHP** [comment]
yeah let's get this in npm! You still around on github? Would you be willing to review PRs? Or let me be a collaborator?

**Comment by matthew-a-thomas** [comment]
@Jakehp I'm happy to let other people join the fun! You should be a collaborator now
